python/frontend/cirrus/tests/preprocessing_test.py

```diff
@@ -0,0 +1,282 @@
+""" Tests for the preprocessing module for Cirrus """
```

Comment:
This test doesn't seem runnable. I would create a main that runs all the tests here.

Also add instructions on how to run this. E.g, where is the data used for the test.